app/controllers/candidate_interface/other_qualifications/details_controller.rb

```diff
@@ -1,106 +1,99 @@
 module CandidateInterface
   class OtherQualifications::DetailsController < OtherQualifications::BaseController
     def new
-      @wizard = wizard_for(current_step: :details)
-      @wizard.qualification_type ||= params[:qualification_type]
-      @wizard.initialize_new_qualification(
+      @form = form_for(current_step: :type)
```

Comment:
I think it might be easier to follow the form initialization logic if each action instantiated the form directly in the state that it's after, rather than via the `form_for` method.  That might make it easier to spot further refinements to the form object's API too.

For example - is it possible to move from an options hash to keyword arguments?

Reply:
`form_for` doesn't really reduce the LOC count much (if at all) and I think it's clearer